components/org.wso2.carbon.identity.api.user.biometric.device/org.wso2.carbon.identity.api.user.biometric.device.common/src/main/java/org/wso2/carbon/identity/api/user/biometric/device/common/util/HandleException.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.carbon.identity.api.user.biometric.device.common.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.api.user.common.error.APIError;
+import org.wso2.carbon.identity.api.user.common.error.ErrorResponse;
+import org.wso2.carbon.identity.application.authenticator.biometric.device.handler.exception.BiometricDeviceHandlerClientException;
+import org.wso2.carbon.identity.application.authenticator.biometric.device.handler.exception.BiometricdeviceHandlerServerException;
+import org.wso2.carbon.user.api.UserStoreException;
+import java.io.UnsupportedEncodingException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.security.SignatureException;
+import java.sql.SQLException;
+import javax.mail.AuthenticationFailedException;
+import javax.ws.rs.core.Response;
+
+/**
+ * The class which handles API errors.
+ */
+public class HandleException {
```

Comment:
Check code formatting.

Also, what is the reason for separating utilities into specific classes? Can we use a single util class?

For example, calling the static method  `handleException` needs to be done in the following manner.
```
HandleException.handleException(...)
```

Having a single util class would improve the readability, and separation of components.
```
BiometricDeviceUtils.handleException(...)
```
```